assets/js/googlesitekit/api/index.test.js

```diff
@@ -100,7 +100,7 @@ describe( 'googlesitekit.api', () => {
 		it( 'should send query string params from data params', async () => {
 			fetch
 				.doMockIf(
-					/^\/google-site-kit\/v1\/core\/search-console\/data\/search/
+					/\/google-site-kit\/v1\/core\/search-console\/data\/search/
```

Comment:
This is less strict than before - why did you remove the `^`?

Reply:
Sorry, that was from an unrelated change/testing of something else and I forgot to restore it. Thanks for catching it, I've reverted that one! 👍